### ISSUE TITLE
PianoRoll: modify default behaviour for volume change on selections

### ISSUE THREAD

**Issue by superpaik** [LMMS/lmms#6681]
### Enhancement Summary
Modify the default behaviour of "PianoRoll change volume for selections".
Now, when you try to change the volume of a selection (with mouse left-button) all volumes are first set to default value (100), loosing the original values.
The propossed behaviour is to change the volume of each note from its own value, not reseting them; and adding an additional action to reset values (like mouse middle-button) to default value.

#### Justification
This behaviour is useful when the user wants to change the volume of the clip (or some notes) but keeping the relative volume between them, which I think it's the expected behaviour for this action.

#### Mockup
First, moving volume level with mouse left-button. Second, reset values with mouse middle-button.

![PianoRoll-Volume](https://user-images.githubusercontent.com/68785450/228283520-b657afed-b8f4-43d5-a2de-e10dbb8bb004.gif)

**Comment by hberg137** [comment]
Hey, is this a feature that you worked on or one that you're looking for someone else to implement? If it's the latter case, I'd like to try and tackle this.

**Comment by superpaik** [comment]
I have it almost ready. I just wanted to know what's the opinion of the community before creating a PR because it's a change in the actual behavior.

**Comment by hberg137** [comment]
Ah ok, gotcha.  If you double click one of the sliders and enter a value with multiple notes selected, does it still update all of the notes selected to have that set value, or does this change that behavior? I think this is a good feature unless it changes the behavior that I just mentioned when entering a value for multiple notes.

**Comment by superpaik** [comment]
No, I'm not changing that behaviour. Only when sliders are moved by mouse. The other actions stay the same.
